infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```diff
@@ -156,7 +162,7 @@ param parPrivateDnsZones array = [
 param parVpnGatewayConfig object = {
   name: '${parCompanyPrefix}-Vpn-Gateway'
   gatewayType: 'Vpn'
-  sku: 'VpnGw1'
+  sku: 'VpnGw1AZ'
```

Comment:
I think we would want to have this dependent on whether av zones are specified or not, rather than hard coded.

Reply:
will look more into this

Reply:
I am going to revert the param files to what they were and let the customer decide to update the SKU. I have added a line in the README for hub networking to specify that you need to select a zonal SKU